src/js/views/patients/worklist/action_views.js

```diff
@@ -59,6 +71,12 @@ const ActionItemView = View.extend({
   onClickPatient() {
     Radio.trigger('event-router', 'patient:dashboard', this.model.get('_patient'));
   },
+  onClickSelect() {
+    const isSelected = this.model.get('isSelected');
```

Comment:
if we pass the state down to the child we can possibly just do `listState.setSelected(this.model);` here... and I think we need to make sure we're not preventing the checkbox click from checking the box on the trigger definition `preventDefault: false`?

Reply:
Then `setSelected` on the state model can determine the type.. and handle the without/push stuff in the state model.  Then perhaps we could add a `listState.isSelected(this.model)`  It also might be cheaper to store these as `[{ 'id1': true, 'id2': false }]` so that each row doesn't have to loop over the array.. isSelected just becomes `selectedFoo[this.model.id]` and the values are either true or falsy (false/null)